eng/jobs/bash-build.yml

```diff
@@ -22,24 +22,24 @@ jobs:
   - job: ${{ parameters.displayName }}
     displayName: ${{ parameters.displayName }}
     pool:
-      ${{ if and(eq(parameters.displayName, 'Build_Linux_x64_FreeBSD'), ne(variables['System.TeamProject'], 'public')) }}:
+      ${{ if and(eq(parameters.displayName, 'Build_FreeBSD_x64'), ne(variables['System.TeamProject'], 'public')) }}:
```

Comment:
(Up on line 8)  does `osGroup` need to be changed to FreeBSD for this group?

Reply:
OSGroup does need to be FreeBSD, but the value in this file is only the default--it's already overridden here:

https://github.com/dotnet/core-setup/blob/c5f6ce6670ba67cf1e5aaec87c1053aa4fc62bf9/azure-pipelines.yml#L134-L138